solo-chains/runtime/dancelight/src/bridge_to_ethereum_config.rs

```diff
@@ -158,4 +178,71 @@ mod benchmark_helper {
             RuntimeOrigin::from(pallet_xcm::Origin::Xcm(location))
         }
     }
+
+    impl snowbridge_pallet_inbound_queue::BenchmarkHelper<Runtime> for EthSystemBenchHelper {
+        fn initialize_storage(beacon_header: BeaconHeader, block_roots_root: H256) {
+            let submit_message = snowbridge_pallet_inbound_queue_fixtures::register_token::make_register_token_message();
+            let envelope: Envelope = Envelope::try_from(&submit_message.message.event_log).unwrap();
+
+            Channels::<Runtime>::set(
+                envelope.channel_id,
+                Some(Channel {
+                    agent_id: Default::default(),
+                    para_id: Default::default(),
+                }),
+            );
+
+            EthereumBeaconClient::store_finalized_header(beacon_header, block_roots_root).unwrap();
+        }
+    }
+
+    pub struct DoNothingMessageProcessor;
+
+    impl MessageProcessor for DoNothingMessageProcessor {
+        fn can_process_message(_: &Channel, _: &Envelope) -> bool {
+            true
+        }
+
+        fn process_message(_: Channel, _: Envelope) -> Result<(), sp_runtime::DispatchError> {
+            Ok(())
+        }
+    }
+}
+
+#[cfg(test)]
+mod test_helpers {
+    use snowbridge_core::inbound::{Log, Proof, VerificationError, Verifier};
+
+    pub struct MockVerifier;
+
+    impl Verifier for MockVerifier {
+        fn verify(_: &Log, _: &Proof) -> Result<(), VerificationError> {
+            Ok(())
+        }
+    }
+}
+
+impl snowbridge_pallet_inbound_queue::Config for Runtime {
+    type RuntimeEvent = RuntimeEvent;
+    #[cfg(not(test))]
+    type Verifier = EthereumBeaconClient;
+    #[cfg(test)]
+    type Verifier = test_helpers::MockVerifier;
+    type Token = Balances;
+    type XcmSender = DoNothingRouter;
+    type GatewayAddress = EthereumGatewayAddress;
+    type MessageConverter = DoNothingConvertMessage;
```

Comment:
Same thing here, let's add  TODOs for in the future modify this whenever we want to support more messages

Reply:
*fixed*